applications/GeoMechanicsApplication/geo_mechanics_application_constants.h

```diff
@@ -0,0 +1,87 @@
+// KRATOS___
```

Comment:
Just mentioning that I like this idea of having the required constants in a separated file rather than adding them in the application ones.